lib/calendarium-romanum/cli.rb

```diff
@@ -115,5 +123,26 @@ def die!(message, code=1)
       STDERR.puts message
       exit code
     end
+
+    private
+
+    def print_single_date(date, sanctorale)
+      calendar = Calendar.for_day(date, sanctorale)
```

Comment:
It's inefficient to create a new `Calendar` instance for each day to print. Instead, use one `PerpetualCalendar` instance for all days.